packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.jsx

```diff
@@ -80,6 +90,61 @@ MultiShippingItemAttributes.propTypes = {
     includeQuantity: PropTypes.bool
 }
 
+// Separate component for address form to properly use hooks
+const AddressFormComponent = ({item, index, onSubmit, onCancel}) => {
+    const form = useForm({
```

Comment:
Can we move this up to MultiAddress component?the submission logic stays in MultiAddress. This func only needs to take form as prop